assets/base.css

```diff
@@ -2491,10 +2491,7 @@ details-disclosure > details {
 .content-container:after {
   content: '';
   position: absolute;
-  width: calc(var(--text-boxes-border-width) * 2 + 100%);
-  height: calc(var(--text-boxes-border-width) * 2 + 100%);
-  top: calc(var(--text-boxes-border-width) * -1);
-  left: calc(var(--text-boxes-border-width) * -1);
+  inset: calc(var(--text-boxes-border-width) * -1);
```

Comment:
Weren't we a bit worried about using `inset` when it's no supported on 14 for Safari desktop and mobile 🤔 
Mainly mobile. I was trying to find the percentage uses of each version but not finding anything so far.

Reply:
https://caniuse.com/?search=inset

You can check here for usage stats for unsupported versions by hovering.  It's a total of 3.5%.

If this is not the sort of thing that will totally break the site on those versions, we can allow them to exist as a progressive enhancement for browsers that do support them.

Reply:
I prefer to go by the `caniuse` stats because they're accounting for actual usage in the wild - on the web.

Reply:
Also if I interpret those correctly that 3.5% is mobile and desktop safari combined and any versions between 3.2 and 14/14.4, meaning some of that percentage could be well outside the last 2 major versions anyway.